muxplore/main.c

```diff
@@ -2287,6 +2325,7 @@ int main(int argc, char *argv[]) {
     lv_indev_drv_register(&indev_drv);
 
     pthread_join(gen_item_thread, NULL);
+    init_footer_elements();
```

Comment:
This has to happen after items have been generated but before it navigates back to where the user left off in the list in order to generate the glyph icons in the footer timely.  Reason being is when the navigation happens it wait for ui changes to finish to make sure it can get the label text out of the current item.